UPDATING.md

```diff
@@ -0,0 +1,54 @@
+# Updating models
+
+A convenience script has been provided, `./update-models.sh`.
+
+The process that's been automated here is as follows:
+
+```bash
+# install and get the generator into place
+git clone https://github.com/openactive/models-lib
```

Comment:
Is it worth cloning one level up to avoid filling the  `models-php` folder? 🤔

Reply:
I was thinking maybe cloning into vendor dir, but feel like that may be a little too composer controlled.

Reply:
That could work 👌